docs/packages/pkg/FastNoiseSIMD.rst

```diff
@@ -0,0 +1,22 @@
+.. spelling::
+
+    FastNoiseSIMD
+
+.. index:: math ; FastNoiseSIMD
+
+.. _pkg.FastNoiseSIMD:
+
+FastNoiseSIMD
+=============
+
+-  `Original <https://github.com/Auburns/FastNoiseSIMD>`__
+-  `Official <https://github.com/caseymcc/FastNoiseSIMD>`__
```

Comment:
Can you explain the original/official part? Do you want me to create https://github.com/hunter-packages fork?

Reply:
The original was a macro base SIMD version, I modified it heavily to templates and altered its interface. I believe the original version is now defunct.

Reply:
> I modified it heavily to templates and altered its interface

So they are not compatible? Would it be better to rename the project and introduce completely new API? You can leave the "Originally based on ... work" note somewhere.